sdk/core/System.ClientModel/src/Message/PipelineMessage.cs

```diff
@@ -113,14 +113,13 @@ public CancellationToken CancellationToken
     /// complete, and prior to the call to
     /// <see cref="ClientPipeline.Send(PipelineMessage)"/>.
     /// </remarks>
-    public void Apply(RequestOptions options)
+    public void Apply(RequestOptions? options)
```

Comment:
I think it would be good to update XML docs to say what happens when null is passed

Reply:
Addressed.  Thanks!